WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -317,6 +317,12 @@ private void validateAndStyleUsername(Collection<String> usernames, final Valida
         }
 
         if (usernamesToCheck.size() > 0) {
+
+            if (!NetworkUtils.checkConnection(getActivity())) {
```

Comment:
The network check here seems to have the side effect of trying it even on device rotation, every time popping up a being-offline Toast even though the user didn't try to actually do anything.

Instead of having the check here, may I suggest moving the network check inside the `public void send()` method, [right before the SEND button disabling](https://github.com/wordpress-mobile/WordPress-Android/pull/4335/files#diff-fdb8073841701f644a4de38798d117c9R426)? I might be wrong with the suggestion so, please give it a shot or perhaps come up with a more correct solution. Thanks!